File: backend/src/main/java/com/pequenospasos/backend/entity/Educador.java
package com.pequenospasos.backend.entity;

import com.pequenospasos.backend.enums.Role;
import jakarta.persistence.DiscriminatorValue;
import jakarta.persistence.Entity;

@Entity
public class Educador extends Usuario {

    public Educador() {
        super();
        this.setTipoUsuario(Role.EDUCADOR);
    }

    public Educador(String nombre, String apellidos, String email, String password, String telefono) {
        super(nombre, apellidos, email, password, telefono, Role.EDUCADOR);
    }
}

File: mobile-app/pequenosPasos/app/src/main/java/com/example/pequenospasos/data/model/Educador.kt
package com.example.pequenospasos.data.model

data class Educador(
    val id: Long,
    val nombre: String = "",
    val apellidos: String = "",
    val email: String = "",
    val password: String = "",
    val telefono: String = "",
    val tipoUsuario: String = "EDUCADOR"
)